### PR TITLE
fix: Crash issue on android

### PR DESCRIPTION
**Pull request by aronshamash**
- Fix issue where android app crashes when first article in Comment section
- https://nidigitalsolutions.jira.com/browse/TNLT-7397

### Before
https://user-images.githubusercontent.com/1736782/114694169-2f17e780-9d12-11eb-951e-2cb2e6b87913.mov

### After
https://user-images.githubusercontent.com/1736782/114696551-b1a1a680-9d14-11eb-996a-f7e5d83857b8.mov

### CODE REVIEW

File: packages/article-summary/src/article-summary.js [hunk 1]
```diff
@@ -71,12 +71,10 @@ function ArticleSummary(props) {
   const { bylineOnTop = false } = bylineProps || {};
 
   const byline = bylineProps ? (
-    <Text>
-      <ArticleSummaryByline
```
Comment (jonnyandrew) [inline packages/article-summary/src/article-summary.js:75]: Hey @aronshamash just to help me learn more about RN, how does this change resolve the crash? What was the purpose of the `<Text>`?

Comment (aronshamash) [reply packages/article-summary/src/article-summary.js:75]: The <Text> was there before and should not have been. I think having a <View> inside and <Text> was causing the issue.

**Review by marionoratoRN — APPROVED** [review]
LGTM

**Review by kamiranoff — APPROVED** [review]

**Comment by kamiranoff** [comment]
LGTM!

**Pull request merged**